config/routes.rb

```diff
@@ -15,7 +15,10 @@
       post '/login', to: 'sessions#create'
       post '/password_update', to: 'passwords#update'
 
-      resource :users
+      resource :users do
+        get '/all', to: 'users#index'
```

Comment:
I don't think we need that way because have CRUD

Reply:
fix